### PR TITLE
Add org.dbgate.DbGate

### PR DESCRIPTION
**Pull request by janproch**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by TingPing** [comment]
You should create a working package before opening a request here...

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "org.dbgate.DbGate",
+  "command": "electron-wrapper",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
```
Comment (hfiguiere) [inline manifest.json:5]: this runtime is EOL.

Comment (janproch) [reply manifest.json:5]: Changed to 22.08, thanks

**Comment by janproch** [comment]
> You should create a working package before opening a request here...

Sorry, I followed and probably misunderstood instructions here https://github.com/flathub/flathub/wiki/App-Submission

Should I simple commit flatpak package into this pull request?

**Comment by TingPing** [comment]
> Should I simply commit flatpak package into this pull request?

I just mean update this PR with a fully functioning manifest.

**Comment by hfiguiere** [comment]
> > You should create a working package before opening a request here...
> 
> Sorry, I followed and probably misunderstood instructions here https://github.com/flathub/flathub/wiki/App-Submission

you don't even tell how to build your package. When it said "Add your app’s manifest to the branch, commit it and push the commit" it mean just that. It implies the manifest is working.

**Comment by janproch** [comment]
This manifest is working, I built the flatpak single-file bundle using it (more exactly it is output of electron flatpak builder). I didn't know anything about flatpak, so I supposed it will work similarily as snap. But I see I must study it a bit more deeply.

**Comment by janproch** [comment]
I pushed updated manifest, now building flatpak from manifest works, but there is still problem in bundled app, when bundling with flatpak

File: org.dbgate.DbGate.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+base-version: '22.08'
+command: dbgate
+rename-icon: dbgate
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  # allow fork subprocess
+  - --socket=system-bus
```
Comment (hfiguiere) [inline org.dbgate.DbGate.yml:15]: you need to specify a dbus name here `system-bus` is not allowed.

**Comment by hfiguiere** [comment]
bot, build org.dbgate.DbGate

File: org.dbgate.DbGate.yml (hunk 2 above)
Comment (hfiguiere) [reply org.dbgate.DbGate.yml:15]: This is what is causing the build failure.

**Comment by rugk** [comment]
Ref https://github.com/flathub/flathub/pull/5240 for new PR BTW

**Pull request closed without merging**